astropy/units/format/generic.py

```diff
@@ -30,7 +30,7 @@ def _to_string(cls, unit):
     if isinstance(unit, core.CompositeUnit):
         parts = []
 
-        if cls._show_scale and unit.scale != 1:
```

Comment:
Removing this means I can refactor `to_string` elsewhere. But here really it just serves to make everything else independent of the deprecated `Unscaled` class.

Reply:
My understanding of a separate deprecatation PR would be to have one just with the `@deprecated` flag in (and some followups like removed tests). Is there a specific reason to have other code changes here?

Reply:
I think it is OK, as here it really helps to ensure that once the deprecation period is over, all one has to do is to remove the class (one could argue about the test, but since I just added it, I think it is fine to remove it already!).

Reply:
OK, thanks.